tools/plugins/com.liferay.ide.project.ui/plugin.xml

```diff
@@ -226,6 +226,16 @@
             %import.binary.plugins.wizard.description
          </description>
       </wizard>
+      <wizard
+            category="com.liferay.ide.projects.import"
+            class="com.liferay.ide.project.ui.workspace.LiferayWorkspaceImportWizard"
+            icon="icons/e16/import.png"
```

Comment:
Lets just use the same icon for new "liferay workspace" wizard for the input.  No need to use the other "import" icon.